python/hail/methods/statgen.py

```diff
@@ -1689,57 +1675,114 @@ def pc_relate(dataset, k, maf, block_size=512, min_kinship=-float("inf"), statis
        PCRelate are often too noisy to reliably distinguish these pairs from
        higher-degree-relative-pairs or unrelated pairs.
 
+    The resulting table has the first 3, 4, 5, or 6 fields below, depending on
+    the `statistics` parameter:
+
+     - `i` (``col_key.dtype``) -- First sample. (key field)
+     - `j` (``col_key.dtype``) -- Second sample. (key field)
+     - `kin` (:py:data:`.tfloat64`) -- Kinship estimate, :math:`\\widehat{\\phi_{ij}}`.
+     - `k2` (:py:data:`.tfloat64`) -- IBD2 estimate, :math:`\\widehat{k^{(2)}_{ij}}`.
+     - `k0` (:py:data:`.tfloat64`) -- IBD1 estimate, :math:`\\widehat{k^{(0)}_{ij}}`.
+     - `k1` (:py:data:`.tfloat64`) -- IBD0 estimate, :math:`\\widehat{k^{(1)}_{ij}}`.
+
+    Here ``col_key`` refers to the column key of the source matrix table,
+    and ``col_key.dtype`` is a struct containing the column key fields.
+
+    There is one row for each pair of distinct samples (columns), where `i`
+    corresponds to the column of smaller column index. In particular, if the
+    same column key value exists for :math:`n` columns, then the resulting
+    table will have :math:`\\binom{n-1}{2}` rows with both key fields equal to
+    that column key value. This may result in unexpected behavior in downstream
+    processing.
+
     Parameters
     ----------
-    ds : :class:`.MatrixTable`
-        A variant-keyed :class:`.MatrixTable` containing
-        genotype information.
-    k : :obj:`int`
-        The number of principal components to use to distinguish ancestries.
+    call_expr : :class:`.CallExpression`
+        Entry-indexed call expression.
     maf : :obj:`float`
```

Comment:
rename to min_individual_af

Reply:
Yeah it's thrown me off a few times that this is not a cohort MAF, but individual. Would be good to make that distinction so people don't think they filtered their dataset more than they did

Reply:
`min_individual_maf` since it's minor allele freq